internal/queries/queries.sql

```diff
@@ -95,6 +95,13 @@ SELECT
     i.indisunique AS index_is_unique,
     COALESCE(parent_c.relname, '')::TEXT AS parent_index_name,
     COALESCE(parent_namespace.nspname, '')::TEXT AS parent_index_schema_name,
+    (
+        SELECT ARRAY_AGG(att.attname ORDER BY indkey_ord.ord)
```

Comment:
I almost wonder if it's worth a comment on this block here that this is just `i.indkey.map(attnum -> column_name(attnum))` but in sql.

Reply:
Might leave out the comment, since it's probably best to read from the docs on it. Pretty nifty SQL function!